### PR TITLE
Borer lewds

### PR DESCRIPTION
**Pull request by ktccd**
[Changelogs]: # (Please make a changelog if you're adding, removing or changing content that'll affect players. This includes, but is not limited to, new features, sprites, sounds; balance changes; map edits and important fixes)

:cl: ktccd
add: Borers can now inject aphrodisiac and anaphrodisiac.
tweak: Borer chemicals now sends flavour text to host when injected into them.
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) 
I added the chems before, but TG kept removing them. Now that TG finally gave up, I felt like it wasn't a meaningless effort to re-add them.
Also added flavour text to the chemicals, because it felt a bit weird how you could inject 40u of bicaridine into a host and they had no idea what's going on or what the borer is doing to them.

### CODE REVIEW

File: code/game/gamemodes/miniantags/borer/borer_chemicals.dm [hunk 1]
```diff
@@ -1,59 +1,82 @@
...
 	var/chemname
 	var/chem_desc = "This is a chemical"
+	var/chem_message //Text sent to the host when injecting chemicals
 	var/chemuse = 30
 	var/quantity = 10
 
 /datum/borer_chem/epinephrine
 	chemname = "epinephrine"
 	chem_desc = "Stabilizes critical condition and slowly restores oxygen damage. If overdosed, it will deal toxin and oxyloss damage."
+	chem_message = "<span class='notice'>You feel a surge of epinephrine flooding your bloodstream!</span>"
```
Comment (LetterJay) [inline code/game/gamemodes/miniantags/borer/borer_chemicals.dm]: I feel like this one is too specific.

Comment (ktccd) [reply code/game/gamemodes/miniantags/borer/borer_chemicals.dm]: I gotta agree that it felt a bit... weird. But I'm having trouble describing epinephrine without making it sound like meth or something like that. I thought of writing something about stablizing or maybe about the oxygen effect, but truth be told, epinephrine does almost nothing if you're not in crit...
I guess IRL you'd feel a rush of adrenaline, but adrenaline is just another name for it :S.

Comment (ktccd) [reply code/game/gamemodes/miniantags/borer/borer_chemicals.dm]: I think I got a replacement in mind, gimme a bit and I'll update this.

**Comment by ktccd** [comment]
@LetterJay How's that?

**Pull request merged**